native/acter/api.rsh

```diff
@@ -1115,6 +1115,43 @@ object MsgContent {
 
     /// the list of url previews 
     fn url_previews() -> Vec<UrlPreview>;
+
+    /// available when user changed his profile
+    fn profile_change() -> Option<ProfileChange>;
+
+    /// available when a user changed another user
+    fn membership_change() -> Option<MembershipChange>;
+}
```

Comment:
I'd rather not plug these into `MsgContent`, they aren't . How about we have them on the activity / notifications & RoomEvent directly - these aren't "messages", so "message content" is not the place  I'd look for that at.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/2751/commits/a085729d8603818c361be06ab1978f05f1a0fff0.
Renamed `msg_content()` API to `message()` in order to highlight it is room message and distinguish it from room state events.
So changed some flutter files for api change.